lang/vi_vn.lang

```diff
@@ -0,0 +1,954 @@
+value.SHADOW_BIAS.0=Hướng Ánh sáng
+value.SHADOW_BIAS.1=Bề mặt Bình thường
+
+option.SHADOW_PIXEL=Bóng Pixel Hóa
+option.SHADOW_PIXEL.comment=Áp dụng hiệu ứng pixel hóa trên bóng.
+value.SHADOW_PIXEL.0=TẮT
+
+#Cấu hình Ánh sáng Màu Khối
+option.MCBL_ANTI_BLEED=Chống Phai màu
+option.MCBL_ANTI_BLEED.comment=Giảm sự lan truyền màu ánh sáng khối không mong muốn. §c[-]§rCó thể khiến lỗi undersampling trở nên dễ nhận thấy hơn.
+option.MCBL_LEGACY_COLOR=Màu Cổ điển
+option.MCBL_LEGACY_COLOR.comment=Sử dụng phương pháp cũ để tính toán màu ánh sáng khối.
+
+#Vật liệu
+option.ADVANCED_MATERIALS=Vật liệu Nâng cao
+option.ADVANCED_MATERIALS.comment=Kích hoạt việc sử dụng bản đồ phản chiếu và bình thường, cho phép hiển thị chi tiết hơn trên nhiều vật thể. §e[]§rTùy chọn này yêu cầu gói tài nguyên chứa bản đồ phản chiếu và/hoặc bản đồ bình thường, tắt nếu không sử dụng. §e[]§rCác tùy chọn yêu cầu vật liệu nâng cao được chỉ định bởi *, thay đổi các tùy chọn này không ảnh hưởng đến hình ảnh hoặc hiệu suất trong khi vật liệu nâng cao bị vô hiệu.
+
+option.MATERIAL_FORMAT=Định dạng Vật liệu*
+option.MATERIAL_FORMAT.comment=Xác định định dạng bản đồ phản chiếu và bản đồ bình thường được sử dụng bởi gói tài nguyên. §e[*]§rCác nghệ sĩ texture thường chỉ ra định dạng nào được sử dụng trong gói tài nguyên của họ, sử dụng định dạng sai có thể dẫn đến kết quả kỳ lạ.
+value.MATERIAL_FORMAT.0=SEUS/Old PBR
+value.MATERIAL_FORMAT.1=labPBR 1.3
+
+option.SSS=Lan truyền Dưới bề mặt*
+option.SSS.comment=Cho phép ánh sáng xuyên qua và lan truyền qua các khối. §a[+]§rHiệu ứng này được hưởng lợi từ TAA.
+
+option.BASIC_SSS=Lan truyền Cơ bản
+option.BASIC_SSS.comment=Kích hoạt phiên bản đơn giản của lan truyền dưới bề mặt đối với một số khối.
+
+option.EMISSIVE=Phát quang
+option.EMISSIVE.comment=Cho phép một số bề mặt sáng hơn.
+value.EMISSIVE.0=§cTẮT
+value.EMISSIVE.1=Chỉ Vật liệu Nâng cao*
+value.EMISSIVE.2=Toàn bộ
+
+option.REFRACTION=Khúc xạ
+option.REFRACTION.comment=Kích hoạt sự méo mó phía sau bề mặt trong suốt có bản đồ bình thường. §c[-]§rBề mặt trong suốt sẽ bị méo mó. §c[-]§rMột số tính năng khác có thể không hoạt động đúng cách với hiệu ứng này được kích hoạt.
+value.REFRACTION.0=TẮT
+value.REFRACTION.1=Chỉ Nước
+value.REFRACTION.2=Toàn bộ
+
+option.ALBEDO_BALANCING=Cân bằng Albedo
+option.ALBEDO_BALANCING.comment=Giảm độ sáng của các bề mặt màu sáng rực rỡ.
+
+option.ALPHA_BLEND=Pha Trộn Alpha
+option.ALPHA_BLEND.comment=Xác định cách thực hiện pha trộn trong suốt.
+value.ALPHA_BLEND.0=Gamma
+value.ALPHA_BLEND.1=Tuyến tính
+
+option.ENTITY_FLASH=Chớp sáng Sinh vật
+option.ENTITY_FLASH.comment=Kích hoạt hiệu ứng chớp sáng cho sinh vật bị hại và creeper gần đó.
+
+#Phản chiếu & Bóng loáng
+option.REFLECTION=Phản chiếu
+option.REFLECTION.comment=Kích hoạt phản chiếu trên nước và bề mặt trong suốt.
+value.REFLECTION.0=§cTẮT
+value.REFLECTION.1=Chỉ Bầu trời & Phản chiếu Đặc biệt
```

Comment:
based on google translate, "Đặc biệt" translates to "special".
google translate suggests "gương soi"

Reply:
No change. Should leave it as is, admin